src/ceph-volume/ceph_volume/tests/conftest.py

```diff
@@ -263,6 +264,10 @@ def device_info_not_ceph_disk_member(monkeypatch, request):
     monkeypatch.setattr("ceph_volume.util.device.disk.blkid",
                         lambda path: {'PARTLABEL': request.param[1]})
 
+@pytest.fixture
+def patched_get_block_devs_lsblk():
```

Comment:
Is there an easy way to add a test for `get_block_devs_lsblk`?

Reply:
Sure, the question is just what do we want to test? It's really only the parsing, which is a split on whitespaces. Will add it...